### ISSUE TITLE
Nan value for loss functions

### ISSUE THREAD

**Issue by hhosseinian** [hhosseinian/GenerativeAI_Paintings#2]
1. Check the training rates
2. check the data loader
3. check the models arch. and optimizers
